src/test/run-selftest-pg

```diff
@@ -30,6 +29,18 @@ PSQL=psql
 findpg pg_ctl
 PGCTL=pg_ctl
 
+if [ "$(which $PGCTL)" == "" ]; then
+    findpg pg_ctlcluster
+    PGCTL=pg_ctlcluster
+
+    if [ "$(which $PGCTL)" == "" ]; then
+        echo "Could not find pg_ctl or pg_ctlcluster"
+	exit 1
```

Comment:
mixed use of space/tab: (`exit 1` has leading tab instead of spaces)

Reply:
Gah! Sorry about that